### DISCUSSION TITLE
+9️⃣ Add Klarna gateway support

### DISCUSSION THREAD

**Discussion by remcotolsma** [pronamic/wp-pronamic-pay#117] [Ideas]
> Ik heb laatst een parasol gekocht en gekozen voor de achter betalen betaalmethode via Klarna. Werkt goed en Klarna heeft een nette interface, misschien overwegen om t.z.t. op te nemen in de Pronamic iDEAL plugin?

Several people (9+) have indicated that they are interested via https://www.pronamic.nl/klarna-voor-wordpress/. It is not clear whether these people are interested in a direct integration with Klarna or via a payment provider such as Mollie. This discussion is about a direct integration with Klarna, but we van also investigate support for Klarna via other payment providers.

## Links

- https://developers.klarna.com/en/nl/kpm/checkout-api

## Internal Links

- **Basecamp to-do:** https://basecamp.com/1810084/projects/18117394/todos/441873101
- **Basecamp to-do:** https://basecamp.com/1810084/projects/10966871/todos/418804987
- **Pronamic.nl interest submissions** https://www.pronamic.nl/wp-admin/admin.php?page=gf_entries&view=entries&id=36&orderby=0&order=ASC&s=klarna&field_id=4&operator=contains

**Comment by rvdsteege** [comment]
From customer:

> Im using memberpress wordpress plugin. From their support i was referred to you. Do you accept klarna payments via the pronomic pay gateway?

_Internal Help Scout ticket: https://secure.helpscout.net/conversation/1711952508/23098_
